docs/runbook/setting-up-your-repo.md

```diff
@@ -0,0 +1,26 @@
+# Setting up your repository
+
+To run your own version of the NHS Book a virtual visit app, you will need to first create a fork of the repository
```

Comment:
Not sure if we need any guidance on creating a github account and an organisation.. May be too much detail

Reply:
If you try and fork while logged out you'll be taken to the login / signup page so I think we can leave that out but we could link to https://docs.github.com/en/github/setting-up-and-managing-organizations-and-teams to show how to setup an org

Reply:
That's good enough in my opinion.